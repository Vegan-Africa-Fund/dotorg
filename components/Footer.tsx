import {AppBar, Typography, Box} from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(() => ({
    appBar: {
        top: 'auto',
        bottom: 0,
        backgroundColor: "#000",
        color: "#fff"
    },
    span: {
        fontWeight: 800
    }
}))

const Footer = () => {
    const classes = useStyles();
    return (
        <AppBar position="fixed" color="default" className = {classes.appBar}>
            <Box 
            display="flex"
            ml={1}
            fontSize="0.44rem">
                <Box flexGrow={0.9}><Typography>Blobject:<span className={classes.span}>blobject</span> </Typography></Box>
                <Typography><span className={classes.span}>blobject</span>Â©2021</Typography>
            </Box>
        </AppBar>
    )
}

export default Footer