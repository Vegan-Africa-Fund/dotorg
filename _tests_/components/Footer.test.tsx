export {}
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

test("Footer is rendered correctly", () => {
 const footer = renderer.create(<Footer />).toJSON()
 expect(footer).toMatchSnapshot()
})