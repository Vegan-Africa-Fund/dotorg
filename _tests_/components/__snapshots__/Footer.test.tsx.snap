// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is rendered correctly 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorDefault makeStyles-appBar-1 mui-fixed MuiPaper-elevation4"
>
  <div
    className="MuiBox-root MuiBox-root-3"
  >
    <div
      className="MuiBox-root MuiBox-root-4"
    >
      <p
        className="MuiTypography-root MuiTypography-body1"
      >
        Fund:
        <span
          className="makeStyles-span-2"
        >
          ₿0.71522656
        </span>
         
      </p>
    </div>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      <span
        className="makeStyles-span-2"
      >
        bitcoin
      </span>
      mission©2021
    </p>
  </div>
</header>
`;
